qemu/tests/cfg/thin_provisioning_image.cfg

```diff
@@ -0,0 +1,49 @@
+- thin_provisioning_image:
```

Comment:
thin_provisioning test the basic function of the feature with emulated scsi devices.
thin_provisioning_image uses images created with qemu-img